tests/installation/ipxe_install.pm

```diff
@@ -174,7 +174,25 @@ sub run {
 
     poweroff_host;
 
-    #virtualization tests use a static ipxe configuration file in O3
+    # Note:
+    # SLE Micro 6.0 Self-Install image does not directly support pxe boot.
+    # To install it on bare metal machine, firstly bring up a minimum system via ipxe
+    # with this function(eg sle15sp5 gm). But we do not need to finish installation,
+    # booting to sshd-server-started is enough, at which we will have a ssh console
+    # to do latter steps.
+    # Then dd the Self-Install iso to a USB device.
+    # And then boot from the USB, and finish installation with the Self-Install iso.
+    # For more details, refer to poo#151498.
+    # To achieve the first step, in testsuite settings,
+    # - set `IPXE_UEFI`: SLE Micro 6.0+ only officially support uefi boot
+    # - set `USB_BOOT`: a flag to indicate this USB installation method,
+    #                   which stops further installation
+    # - set `MIRROR_HTTP`: the repository to bring up a minimum system(eg sle15sp5 gm)
+    # - do NOT set `AUTOYAST`
+
+    die "Can't set AUTOYAST for usb boot!" if (get_var('AUTOYAST', '') && is_usb_boot);
```

Comment:
Why not put this line and preceding description before ```poweroff_host``` ?

Reply:
You do not need to explain the whole function. You just need to die immediately.